src/pokemon.c

```diff
@@ -0,0 +1,5567 @@
+void sub_02077618(Pokemon * src, BoxPokemon * dest);
+s8 sub_0207762C(Pokemon * mon, int param1);
+s8 sub_02077634(BoxPokemon * boxMon, int param1);
+s8 sub_02077648(u32 monPersonality, int param1);
+void sub_020776B0(Party * party);
+u8 sub_02077758(Party * party, u8 param1);
+void sub_020777B4(Party * party, s32 param1);
+void sub_0207782C(Party * party);
+BOOL sub_020778D8(Pokemon * mon);
+BOOL sub_020778E0(BoxPokemon * boxMon);
+BOOL sub_020778F8(Pokemon * mon);
+BOOL sub_02077900(BoxPokemon * boxMon);
+void SetArceusForm(Pokemon * mon);
+void SetBoxArceusForm(BoxPokemon * boxMon);
+u8 GetArceusItemType(u16 itemHoldEffect);
+int SetGiratinaForm(Pokemon * mon);
+int SetBoxGiratinaForm(BoxPokemon * boxMon);
+void SetShayminForm(Pokemon * mon, int monForm);
+void SetBoxShayminForm(BoxPokemon * boxMon, int monForm);
+void LoadMonLevelUpMoves(int monSpecies, int monForm, u16 * monLevelUpMoves);
+void sub_02077D3C(UnkStruct_0202CC84 * param0, int param1, u16 monSpecies, int param3, int param4, int param5, int param6, int param7);
+void sub_02077E64(Pokemon * mon, UnkStruct_02025E6C * param1, int monPokeball, int param3, int param4, int param5);
+void sub_02077EA4(BoxPokemon * boxMon, UnkStruct_02025E6C * param1, int monPokeball, int param3, int param4, int param5);
+void sub_02077EE4(Pokemon * mon, UnkStruct_02025E6C * param1, int monPokeball, int param3, int param4, int param5);
+void sub_02077EF8(BoxPokemon * boxMon, UnkStruct_02025E6C * param1, int monPokeball, int param3, int param4, int param5);
+void sub_02077F0C(Pokemon * mon, u32 param1, int param2);
+BOOL sub_02077FB4(Pokemon * mon, u8 param1);
+BOOL sub_02077FBC(BoxPokemon * boxMon, u8 param1);
+BOOL sub_02077FE4(u16 monSpecies, int monForm, u8 tmID);
+void CalculateMonAbility(Pokemon * mon);
+void CalculateBoxMonAbility(BoxPokemon * boxMon);
+void sub_020780C4(Pokemon * mon, u32 monPersonality);
+static void LoadMonPersonalData(int monSpecies, PokemonPersonalData * monPersonalData);
+static void LoadMonFormPersonalData(int monSpecies, int monForm, PokemonPersonalData * monPersonalData);
+static void LoadMonEvolutionData(int monSpecies, PokemonEvolutionData * monEvolutionData);
+static void EncryptMonData(void * data, u32 bytes, u32 seed);
+static void DecryptMonData(void * data, u32 bytes, u32 seed);
+static u16 GetMonDataChecksum(void * data, u32 bytes);
+static void * GetBoxMonDataBlock(BoxPokemon * boxMon, u32 personality, u8 dataBlockID);
+static int GetMonFormNarcIndex(int monSpecies, int monForm);
+u32 GetSingleBitMask(int index);
+int GetLowestSetBitIndex(u32 num);
+BOOL sub_02078804(u16 param0);
+BOOL sub_02078838(Pokemon * mon);
+int sub_020788D0(int param0);
+static void sub_02076300(UnkStruct_02008A90 * param0, u16 monSpecies, u8 monGender, u8 param3, u8 monShininess, u8 monForm, u32 monPersonality);
+static u8 sub_020767BC(u16 monSpecies, u8 monGender, u8 param2, u8 monForm, u32 monPersonality);
+
+void ZeroMonData (Pokemon * mon)
```

Comment:
No space between function name and left-paren.

Reply:
Actually this is out of scope, I'll get to running uncrustify sometime

Reply:
The style guide in https://github.com/pret/pokeheartgold/issues/220 currently has the following note, assuming its outdated?:
> [pokeplatinum only] Space between function name and opening parenthesis for function parameters for a function definition (to make it easy to search for function definitions without an IDE, can just search for “MyFunction (“)